source/MRViewer/MRUIStyle.h

```diff
@@ -45,6 +45,39 @@ struct ButtonIconCustomizationParams : public ButtonCustomizationParams
     bool flatBackgroundColor = false;
 };
 
+struct PlotAxis
```

Comment:
I think its better to have structure for single axis, and pass two instances into plot function, one for X and another for Y